ext/civi_mail/settings/Mailing.setting.php

```diff
@@ -389,4 +389,25 @@
     'help_text' => ts('CiviMail records email sent at the frequency you specify. If you set it to 1, it will update the database every time it sends an email. This ensures that emails are not resent if the batch job fails, but this may cause a performance hit, particularly for large jobs.'),
     'settings_pages' => ['mail' => ['weight' => 60]],
   ],
+  'civimail_default_unsubscribe_mode' => [
```

Comment:
@demeritcowboy given this is more about one click rather than the overall unsubcribe should I change this to be like default_oneclick_unsubscribe_mode?

Reply:
That makes sense, although a traditionally more civi-like name would be "do_not_opt_out_on_unsubscribe" (grin).

And then the title and description below.